src/WebJobs.Script.WebHost/WebHostResolver.cs

```diff
@@ -199,23 +202,25 @@ internal static ScriptHostConfiguration CreateScriptHostConfiguration(WebHostSet
             return scriptHostConfig;
         }
 
-        private static void InitializeFileSystem()
+        private static void InitializeFileSystem(WebHostSettings settings)
         {
             if (ScriptSettingsManager.Instance.IsAzureEnvironment)
             {
+                // When running on Azure, we kick this off on the background
+                // Q: why?
```

Comment:
Not sure if the logic here was "run on background when running on Azure" or "run on background if there's work to do", which only happened on Azure. The intention here is not really clear.

Reply:
it just seemed odd to me that this path is running in a task. I wasn't sure why so I put the comment. Still not really sure why it was done like this in the first place :)